### DISCUSSION TITLE
Galera Next.js usa react? É uma outra lib?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1633] [Perguntas e Respostas]
Fala galera, 
boa tarde, desculpe perguntar aqui, mas ainda vou começar meus estudos e ler a documentação do next.js, e estou um tanto em dúvida entre next.js, gatsby, ou usar apenas o react mesmo em nossos projetos aqui na empresa.

Enfim, qual seria a diferença entre next.js e react, viq ue usamo pratimanete a mesma estrutura... muda muito, preciso tomar certos cuidados? a equipe estando familiarizada com react se dara bem com next ou gatsby?

Muito obrigado, abraços!

**Comment by fdaciuk** [comment]
Fala @wilsonneto-dev! Essa talk pode te ajudar a entender melhor essas diferenças:
https://www.infoq.com/br/presentations/o-poder-do-ssr/

=)

**Comment by NathanFRibeiro** [comment]
@wilsonneto-dev, o Next.js é basicamente um framework React que auxilia principalmente no SEO das aplicações, por meio do server side rendering (SSR). Sabendo React, facilmente se adaptará ao Next.js.

Dá uma olhada nesse artigo que é bem esclarecedor: https://blog.rocketseat.com.br/ssr-nextjs-reactjs/

**Comment by wilsonneto-swe** [comment]
Opa, valeu @NathanFRibeiro, sim, fiz alguns cursos sobre next, inclusive no bootcamp da RocketSeat em um dos módulos bônus tinha bastante conteúdo sobre... Já to usando em produção e me ajudar demais no rankeamento do site no google, nos links de shares e senti um ganho legal de performance... (Ao menos no tempo com que os componentes são renderizados na tela).

Valeeu!!
